packages/react-devtools-core/README.md

```diff
@@ -14,35 +14,41 @@ If you are building a non-browser-based React renderer, you can use the backend
 
 ```js
 if (process.env.NODE_ENV !== 'production') {
-  const { connectToDevTools } = require("react-devtools-core");
+  const { initialize, connectToDevTools } = require("react-devtools-core");
 
+  initialize(settings);
   // Must be called before packages like react or react-native are imported
-  connectToDevTools({
-    ...config
-  });
+  connectToDevTools({...config});
 }
 ```
 
 > **NOTE** that this API (`connectToDevTools`) must be (1) run in the same context as React and (2) must be called before React packages are imported (e.g. `react`, `react-dom`, `react-native`).
 
+### `initialize` arguments
+| Argument   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `settings` | Optional. If not specified, or received as null, then default settings are going to be applied: appending component stacks to error messages and displaying inline errors and warnings.<br/>Can be plain object or a Promise that resolves with the plain object with [these boolean fields](https://github.com/facebook/react/blob/20cec76c44f77e74b3a85225fecab5a431cd986f/packages/react-devtools-shared/src/backend/types.js#L531-L536). If Promise rejects, the console will not be patched and some console features from React DevTools will not work. |
```

Comment:
Optional. If not specified, or received as null, then default settings are **used**.

Reply:
Fixed.